core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2525,4 +2525,9 @@ private CarbonCommonConstants() {
    * property which defines the presto query default value
    */
   public static final String IS_QUERY_FROM_PRESTO_DEFAULT = "false";
+
+  /**
+   * property to send load model from coordinator to worker in presto
+   */
+  public static final String CARBON_PRESTO_LOAD_MODEL = "presto.carbondata.encoded.loadmodel";
```

Comment:
Is this a configurable item by user? If not, suggest to move to presto integration module only.
And I think it is better to change to start with "carbondata", but not "presto" since it is a configuration from carbondata

Reply:
its not configurable by user. You are right. I have moved the constant to `CarbonTableConfig` in presto module and renamed it to `carbondata.presto.encoded.loadmodel`